awx_collection/plugins/module_utils/tower_api.py

```diff
@@ -740,3 +743,179 @@ def wait_on_url(self, url, object_name, object_type, timeout=30, interval=10):
     def wait_output(self, response):
         for k in ('id', 'status', 'elapsed', 'started', 'finished'):
             self.json_output[k] = response['json'].get(k)
+
+    def create_schema_nodes(self, response, schema, workflow_id):
```

Comment:
Since we only have one module now can we move this logic into the module instead of the tower_api library?